newIDE/app/src/AssetStore/AssetDetails.js

```diff
@@ -215,9 +215,13 @@ export const AssetDetails = React.forwardRef<Props, AssetDetailsInterface>(
 
     React.useEffect(
       () => {
-        loadAsset();
+        if (!asset) {
+          loadAsset();
+        }
       },
-      [loadAsset]
+      // loadAsset seem unstable so we don't list it and only load when asset is not defined.
```

Comment:
Is there a possibility that loadAsset is not up to date and this effect uses an out-of-date version of the function?

Reply:
oh actually, I don't need that. The check on `!asset` is enough